.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,72 @@
+name: Bug Report
```

Comment:
Although I like the idea of better structured bug reports I'm not sure this set of templates is well suited for the project. I assume this lets you pick a style before proceeding to filing an issue? Is there an escape hatch that allows not using a template?

Reply:
There is a way to still use blank issues, it can be enabled from the `config.yml` file I created. I'll enable it now.